src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RoleAssignmentServiceTest.java

```diff
@@ -26,4 +27,13 @@ void testRASFeignClientTest() {
         assertEquals(HttpStatus.CREATED, responseEntity.getStatusCode());
     }
 
+    @Test
+    void testGetServiceStatus() {
+        Mockito.when(rasFeignClient.getServiceStatus())
+                .thenReturn("Okay!");
+        String serviceStatus = sut.getServiceStatus();
+        assertNotNull(serviceStatus);
+        assertEquals("Okay!", serviceStatus);
```

Comment:
This is perfectly fine, but you could pass "{\"status\":\"UP\"}" as the string to make it look similar to the actual output.

Bootrun RAS and hit localhost:4096 in your browser to see the actual status given by this endpoint.